pkg/testutil/require.go

```diff
@@ -222,7 +224,9 @@ func statefulSetIsReady(ss *appsv1.StatefulSet) bool {
 	return ss.Status.ReadyReplicas == ss.Status.Replicas
 }
 
-func requireDownGradeOptionSet(t *testing.T, sb testenv.DiffingSandbox, b testutil.ClusterBuilder, version string) {
+// TODO we are not using this
```

Comment:
what do you mean we aren't using this? We aren't checking to see if the downgrade option is set before starting an upgrade?

Reply:
We are checking it in the operator code, but we are not using this code in any of the e2e tests.

Reply:
https://github.com/cockroachdb/cockroach-operator/issues/537